src/Sylius/Bundle/CoreBundle/Provider/DefaultTaxZoneProviderInterface.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\CoreBundle\Provider;
+
+use Sylius\Component\Addressing\Model\ZoneInterface;
+
+/**
+ * @author Mateusz Zalewski <mateusz.zalewski@lakion.com>
+ */
+interface DefaultTaxZoneProviderInterface
```

Comment:
IMHO we don't need that "DefaultTax" prefix. `ZoneProviderInterface` is enough. Then in constructor that uses it to get the default tax zone, just use `__construct(ZoneProviderInterface $defaultTaxZoneProvider)`.